Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -2152,6 +2193,27 @@ theorem isLittleO_pi {ι : Type _} [Fintype ι] {E' : ι → Type _} [∀ i, Nor
   exact ⟨fun h i c hc => h hc i, fun h c hc i => h i hc⟩
 #align asymptotics.is_o_pi Asymptotics.isLittleO_pi
 
+lemma eventuallyLE_of_isLittleO_of_isBigO_symm {f₁ f₂ : α → E'} (hf₁ : f₁ =o[l] g)
+    (hf₂ : g =O[l] f₂) : (fun x => ‖f₁ x‖) ≤ᶠ[l] fun x => ‖f₂ x‖ := by
```

Comment:
```suggestion
    (hf₂ : g =O[l] f₂) : ∀ᶠ x in l, ‖f₁ x‖ ≤ ‖f₂ x‖ := by
```
This is defeq, but I find this form more readable. What do you think?

Reply:
Yes, fair enough.